package.json

```diff
@@ -130,7 +130,6 @@
     "selenium-webdriver": "4.0.0-alpha.5",
     "shelljs": "0.8.3",
     "style-loader": "1.0.0",
-    "svg-inline-loader": "0.8.0",
```

Comment:
@SebastienGllmt 
Probably now we want to get rid of  `"react-svg-inline": "2.1.1"` as well

I can fix that if answer is Yes.